locales-pending/fix.ftl

```diff
@@ -0,0 +1,63 @@
+# Strings in this file are not yet final, and thus should not be localised yet.
+
+fix-flow-nav-data-broker-profiles = Data broker profiles
+fix-flow-nav-high-risk-data-breaches = High risk data breaches
+fix-flow-nav-leaked-passwords = Leaked passwords
+fix-flow-nav-security-recommendations = Security recommendations
+fix-flow-data-broker-profiles-start-free-scan-headline = Find out which sites are selling your personal info
+fix-flow-data-broker-profiles-start-free-scan-content-p1 = Before you review your data breaches, would you like us to search for you on 190 data broker sites that may be selling your personal information? 
```

Comment:
I'd prefer not to do that, because then it becomes a huge pain to translate (6 variants for some locales). Honestly, better to be more vague, e.g. "almost 200 hundred", because nobody will go and keep this up to date with the real number…

Reply:
In this case, I think `hundreds` would work just as well.

Reply:
BTW, looks like we used the plural approach on another string :-( 

```
onboarding-get-started-how-it-works-dialog-step1-content = {
    $dataBrokerTotalCount ->
        *[other] With just a few key pieces of information, we’ll search for you in all known data breaches and { $dataBrokerTotalCount } major data broker sites. All users get the first scan free.
}
```

Can we ask content to revisit this, and avoid shooting ourselves in the foot by putting a number that needs to be kept up-to-date? Based on previous experiences, we'll just forget to update it anyway, and this creates extra-work for some languages that is not completely warratend.

Reply:
This isn't scientific or probably a great <kbd>grep</kbd>(tm), but possibly interesting:

```sh
git grep -IPn "\b190\b" -- ':!*.svg' | cat

.env-dist:78:ONEREP_DATA_BROKER_COUNT=190
locales-pending/fix.ftl:8:fix-flow-data-broker-profiles-start-free-scan-content-p1 = Before you review your data breaches, would you like us to search for you on 190 data broker sites that may be selling your personal information?
locales-pending/fix.ftl:41:# $data_broker_count is the number of data brokers scanned monthly (As of adding this, it is 190). This number will always be plural.
locales-pending/fix.ftl:59:# $data_broker_count is the number of data brokers scanned monthly (As of adding this, it is 190). This number will always be plural.
locales-pending/onboarding.ftl:16:#   $dataBrokerTotalCount (number) - number of scanned data broker sites, e.g. 190
locales-pending/onboarding.ftl:62:#   $dataBrokerTotalCount (number) - number of scanned data broker sites, e.g. 190
src/app/(proper_react)/redesign/(authenticated)/user/dashboard/DashboardTopBanner.tsx:52:          data_broker_sites_total_num: 190,
src/app/(proper_react)/redesign/(authenticated)/user/dashboard/DashboardTopBanner.tsx:69:          data_broker_sites_total_num: 190,
src/app/(proper_react)/redesign/(authenticated)/user/dashboard/fix/data-broker-profiles/automatic-remove/page.tsx:30:              data_broker_count: 190,
src/app/(proper_react)/redesign/(authenticated)/user/dashboard/fix/data-broker-profiles/automatic-remove/page.tsx:76:                    data_broker_count: 190,
src/app/(proper_react)/redesign/(authenticated)/user/welcome/Onboarding.stories.tsx:20:      dataBrokerCount={190}
src/app/components/client/ExposuresFilterExplainer.tsx:30:          data_broker_sites_total_num: 190,
src/app/components/client/ExposuresFilterExplainer.tsx:61:          data_broker_sites_total_num: 190,
```

Reply:
As @pdehaan already found out: We are setting `ONEREP_DATA_BROKER_COUNT` as an env variable. It’s not only used for strings, but also calculations on views. I’d expect that variable to not change often. If it does, it will be updated. Nevertheless, if this is still a localization concern because of the additional effort this creates we’ll need to sync with PM & Content.

Reply:
@flozia How do you know it's 190?
I found https://onerep.com/sites-we-remove-from and if I copy-paste+merge+dedupe the "People-search sites" and "Mirror people-search sites" sections, I get 195 results. But not sure if you're using some API or better method of calculating. I guess we could just have a monthly/quarterly calendar reminder or something if it truly is a manual task to count up results and keep them in sync.

Ah, OK, maybe they _do_ have an API for this: https://docs.onerep.com/#tag/Data-Brokers
But I can't ping it directly since I don't have an API key.

Reply:
@pdehaan This is a side-note, but to help lock in the exact number, we're discussing/determining how to support individual logos for each data broker.

Reply:
@flozia @pdehaan As stated else where in this review, I am using the plural fluent format `*[other] {...` and referencing the environment variable. 

One note for @flozia: I ran into a hydration error issue when doing this, so it's still hard-coded, but I added a FIXME comment regarding it for now. (See 6e660ee1c87da0f3c7f9648f7c9868b698bbf957)